src/plot_api/plot_api.js

```diff
@@ -750,6 +751,17 @@ function cleanData(data, existingData) {
             }
         }
 
+        // fix typo in colorscale definition
```

Comment:
Should we maybe put the check for `YIGnBu` in a attribute coercion step instead of in logic code?